_includes/setup/next-steps-win-linux.md

```diff
@@ -0,0 +1,5 @@
+## Next steps
+
+Now, follow our [Getting Started guide](/getting-started/)
+to run your first Flutter app on Android. To develop for iOS in addition to Android,
+follow the [Flutter Setup for Mac](/setup-mac/) instructions.
```

Comment:
This was in response to PR feedback that we needed to clarify that iOS development requires a Mac. Also, in this PR, there's a comment that apparently many folks are running Windows on a Mac, so yeah - not as strange as it seems. Finally, (@mit-mit who feels strongly) the original wording isn't platform-agnostic. " iOS and Android" doesn't apply to Windows and Linux. 

Changed to "iOS or Android" and re-unified these for now.